src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  isAlive() {
+    if (this.health <= 0) {
+      Animal.alive = Animal.alive.filter((animal) => animal !== this);
+    }
+  }
+
+  setHealth(health) {
+    this.health = health;
+  }
```

Comment:
[CODE QUALITY] The `setHealth` method is not necessary since the health property can be directly manipulated within the class. You could directly subtract from the `health` property in the `bite` method, which would simplify the code.